### PR TITLE
add swapdims frontends.torch

### PR DESCRIPTION
**Pull request by PatternFinder**
link to issue: #2705

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py [hunk 1]
```diff
@@ -123,3 +123,55 @@ def test_permute(
...
+    input_dtype, value = dtype_value
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="torch",
+        fn_name="swapdims",
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:173]: please rename this argument to `fn_tree`

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there! Great work so far! Just a small recent change and you should be good to go! As usual, please request my review once you're done! Thanks!

**Review by juliagsy — APPROVED** [review]
Thanks for contributing! Will merge soon!

**Comment by PatternFinder** [comment]
> Thanks for contributing! Will merge soon!

Thank you! see you in other PRs :)

**Pull request merged**